content/showcase/_index.html

```diff
@@ -12,6 +12,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
         <div class="columns is-centered">
             <div class="column is-8">
                 <div class="showcase-gallery columns is-multiline">
+                    {{< showcase-item
+                        src="/img/media/showcase/imvim.jpg"
+                        title="imvim"
+                        caption="<p>\"Improved\" Vi Improved: the c̶u̶r̶s̶e̶d̶ (efficient and intuitive!!) text-editor. Created using Tkinter.</p><p><b>2024 Hackathon Most Masochistic</b></p>"
```

Comment:
Instead of using unicode characters, try using the `<s>` tag to get the strike through. This should be more consistent across browsers and systems.

Reply:
ok no problem will do that now :)